cats/cats-sql/cats-sql.gradle

```diff
@@ -21,27 +21,50 @@ apply plugin: "groovy"
 tasks.compileGroovy.enabled = false
 
 dependencies {
-  spinnaker.group("spockBase")
-  testCompile spinnaker.dependency("groovy")
-
-  compile project(':cats:cats-core')
-  compile project(':cats:cats-redis')
-  compile project(":clouddriver-sql")
-  compile project(":clouddriver-core")
-  compile("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.1")
-  compile("org.jetbrains.kotlinx:kotlinx-coroutines-core-common:1.1.1")
-  compile("org.jetbrains.kotlinx:kotlinx-coroutines-slf4j:1.1.1")
-
-  testCompile "com.h2database:h2:1.4.197"
-
-  testCompile "io.strikt:strikt-core:0.11.5"
-  testCompile "org.assertj:assertj-core:3.9.0"
-  testCompile "org.junit.jupiter:junit-jupiter-api:${spinnaker.version("jupiter")}"
-  testCompile "com.nhaarman:mockito-kotlin:1.5.0"
-
-  testCompile project(':cats:cats-test')
-  testCompile(spinnaker.dependency("korkSqlTest"))
-  testRuntime "org.junit.jupiter:junit-jupiter-engine:${spinnaker.version("jupiter")}"
-  testRuntime "org.junit.platform:junit-platform-launcher:${spinnaker.version("junit5")}"
-  testRuntime "org.junit.vintage:junit-vintage-engine:${spinnaker.version("junitVintage")}"
+  implementation project(":cats:cats-core")
+  implementation project(":cats:cats-redis")
+  implementation project(":clouddriver-core")
+  implementation project(":clouddriver-security")
+  implementation project(":clouddriver-sql")
+
+  compileOnly "org.projectlombok:lombok"
+  annotationProcessor "org.projectlombok:lombok"
+  testAnnotationProcessor "org.projectlombok:lombok"
+
+  implementation "com.fasterxml.jackson.core:jackson-databind"
+  implementation "com.h2database:h2:1.4.197"
+  implementation "com.netflix.spectator:spectator-api"
+  implementation "com.netflix.spinnaker.fiat:fiat-api:$fiatVersion"
+  implementation "com.netflix.spinnaker.fiat:fiat-core:$fiatVersion"
+  implementation "com.netflix.spinnaker.kork:kork-core"
+  implementation "com.netflix.spinnaker.kork:kork-sql"
+  implementation "de.huxhorn.sulky:de.huxhorn.sulky.ulid"
+  implementation "io.github.resilience4j:resilience4j-retry"
+  implementation "io.strikt:strikt-core:0.11.5"
```

Comment:
is there a reason we specify a version here vs. using bom?
 https://github.com/spinnaker/kork/blob/master/spinnaker-dependencies/spinnaker-dependencies.gradle#L53